### PR TITLE
styles: add styling for upgrade page

### PR DESCRIPTION
**Pull request by ashmonty**

### CODE REVIEW

File: views/account/upgrade.handlebars [hunk 1]
```diff
@@ -26,25 +26,62 @@
...
-<style>
-	.tier {
-		padding: 10px;
-		background-color: white;
-		color: black;
-	}
-</style>
\ No newline at end of file
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
```
Comment (mrjvs) [inline views/account/upgrade.handlebars:69]: use `<script defer>` and remove the domContentLoaded event.

File: views/account/upgrade.handlebars [hunk 2]
```diff
@@ -26,25 +26,62 @@
...
+			const selectedTier = form.querySelector('input[type="radio"]:checked').value;
+			form.action = `/account/checkout/${selectedTier}`;
+			form.submit();
+		});
+
+		document.querySelector('form').addEventListener('change', function(e) {
+			e.preventDefault();
+
+			const clickedTierID = e.srcElement.id;
+			const clickedTierName = document.querySelector(`label.tier[for="${clickedTierID}"] .tier-name`).innerHTML;
```
Comment (mrjvs) [inline views/account/upgrade.handlebars:83]: either use innerText or store the name in a data-* attribute. (I prefer the latter)

File: views/account/upgrade.handlebars [hunk 3]
```diff
@@ -26,25 +26,62 @@
...
+			document.querySelector('#submitButton').innerHTML = `Subscribe to ${clickedTierName}`;
```
Comment (mrjvs) [inline views/account/upgrade.handlebars:84]: use innerText

File: views/account/upgrade.handlebars (hunk 2 above)
Comment (mrjvs) [reply views/account/upgrade.handlebars:83]: (also use data attribute instead of quering the for attribute)

**Review by mrjvs — CHANGES_REQUESTED** [review]

**Review by mrjvs — APPROVED** [review]

**Comment by ashmonty** [comment]
when we mergin bois

**Pull request merged**